umm-spec-lib/resources/json-schemas/collection/umm/v1.16.7/umm-search-results-json-schema.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
+      "properties": {
+        "meta": {
+          "$ref": "#/definitions/MetaType"
+        },
+        "umm": {
+          "$ref": "umm-c-json-schema.json"
+        }
+      },
+      "required": ["meta"]
+    },
+    "MetaType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "CMR level metadata about the item found. This represents data not found in the actual metadata.",
+      "properties": {
+        "granule-count": {
+          "description": "A number >= 0 that indicates the number of granules for the collection.",
```

Comment:
I haven't noticed in others, but do we want to specify an integer number?

Reply:
I think it's implied. I haven't changed anything in this file. It's just copied over.